lib/kitchen/directions/bake_link_placeholders.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bake directions for link placeholders
+    #
+    module BakeLinkPlaceholders
+      def self.v1(book:, pantry:)
```

Comment:
`pantry` should be the same as `book.document.pantry` so can probably nix the second argument